src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -307,5 +307,12 @@ trait MarathonConf
     noshort = true,
     hidden = true,
     default = Some(MesosHeartbeatMonitor.DEFAULT_HEARTBEAT_FAILURE_THRESHOLD))
+
+  lazy val drainingTime = opt[Long](
+    "draining_time",
```

Comment:
no, I mean that probably this is what it should be doing :P. I will re-think it a little bit, but I would like to get some feedback before rewriting from @timcharper